content/opportunities/Marketing-Assistant-and-Copywriter.mdx

```diff
@@ -11,43 +11,45 @@ hideApply: true
-* Your "can-do" attitude and commitment to getting the job done will secure you an interview 
+* Write targeted content for websites, blogs, newsletters and other types of marketing and or promotional material
+* Delivering SEM, organic and paid social media campaigns across multiple platforms
+* Assist in the design, creation and rollout of content across multiple social media sites, web, video and traditional PR channels
+* Assist with planning, developing and delivering digital strategies that build brand awareness
+* Create content for social media channels (LinkedIn, Twitter, Facebook, Instagram & TikTok)
+* Work effectively as a part of a team to ensure efficient and effective processes are identified, initiated and undertaken
+* Keep abreast of industry trends and compelling marketing campaigns
+* Improving our ‘Rules to Better’ Marketing, Blogs, Social Media, etc.
+* Reporting on analytics to the Marketing Manager, CEO & Sales team 
+* Event management 
+* Liaising with developers and clients
+
+#### The successful candidate will have 
+
+* The right to live and work in Australia
+* At least 2 years of experience in a Marketing Role
+* A degree in Marketing or equivalent
+* Experience in writing blogs or equivalent
+* Experience in Google and social media paid campaigns
+* Basic knowledge of HTML
+* Fantastic interpersonal and communication skills
+* Self-motivation and high attention to detail
+* Your "can-do" attitude and commitment to getting the job done will secure you an interview
 
 ***
 
 #### About SSW
 
-We deliver scalable and extensible custom solutions with the future in mind. SSW is a great place to work, as we produce world-class modern solutions, open many avenues to bolster your public profile, foster an inclusive and supportive office environment, and much more. 
+Check out this video to find out more about us: [Why work at SSW?](https://www.youtube.com/watch?v=tWuutobJtdg)
+
+We deliver scalable and extensible custom solutions with the future in mind. SSW is a great place to work, as we
+produce world-class modern solutions, open many avenues to bolster your public profile, foster an inclusive and supportive office environment, and much more.
 
-We are committed to increasing our workplace diversity, and we urge people from all walks of life to apply. We encourage expertise, mastery, and respectful discourse. We aim to build you up and hope you'll do the same! Our horizontal organisational structure means that your input will make an impact on the way we do business. 
+We are committed to increasing our workplace diversity, and we urge people from all walks of life to apply. We encourage expertise, mastery, and respectful discourse. We aim to build you up and hope you'll do the same! We will help you grow your community profile and your technical abilities so you can be the best developer you can be. Our horizontal  organisational structure means that your input will make an impact on the way we do business.
```

Comment:
Remove "We will help you grow your community profile and your technical abilities so you can be the best developer you can be." as this is only for developers.